transplant_guide.md

```diff
@@ -104,13 +105,16 @@ Neighborhoods _Elia_
 ![seattle neighborhoods](seattle-neighborhood-map.jpg "Seattle Neighborhoods")  
 
 **Pioneer Square**
-Very eclectic neighborhood. Lots of homeless. The oldest neighborhood in Seattle!
+The oldest neighborhood in Seattle! There are sometimes cheap studios and one bedrooms here, but more are getting renovated into expensive loft-like spaces.
+This is close to the football and baseball stadiums, so sometimes traffic can be bad, and if you have a car and no dedicated parking that can be a big problem. Several of the homeless shelters/missions are in this area, so there are quite a lot of homeless people in the area, which could be nerve-wracking walking home alone at night if you are not moving to Seattle from a city. It is just south of downtown, so walking to class is definitely doable.
+If seeing drug use on the streets near where you live feels scary, this may not be the best neighborhood to look in for you. No real grocery store in this area.
 
 **International District (ID) / Chinatown**
-Info about ID
+The ID used to be called Chinatown, so this is its new politically correct name. It is right next to Pioneer Square, and there is not a super clear line of deliniation between the two neighborhoods, so a lot of the commentary about Pioneer Square applies here.
```

Comment:
Actually, Chinatown is the _new_ name. It used to be a Japanese neighborhood back in the day.